### ISSUE TITLE
"No video could be found in this tweet" even though there definitely is video

### ISSUE THREAD

**Issue by TimothyTRiG** [yt-dlp/yt-dlp#7950]
### DO NOT REMOVE OR SKIP THE ISSUE TEMPLATE

- [X] I understand that I will be **blocked** if I *intentionally* remove or skip any mandatory\* field

### Checklist

- [X] I'm reporting that yt-dlp is broken on a **supported** site
- [X] I've verified that I'm running yt-dlp version **2023.07.06** ([update instructions](https://github.com/yt-dlp/yt-dlp#update)) or later (specify commit)
- [X] I've checked that all provided URLs are playable in a browser with the same IP and same login details
- [X] I've checked that all URLs and arguments with special characters are [properly quoted or escaped](https://github.com/yt-dlp/yt-dlp/wiki/FAQ#video-url-contains-an-ampersand--and-im-getting-some-strange-output-1-2839-or-v-is-not-recognized-as-an-internal-or-external-command)
- [X] I've searched [known issues](https://github.com/yt-dlp/yt-dlp/issues/3766) and the [bugtracker](https://github.com/yt-dlp/yt-dlp/issues?q=) for similar issues **including closed ones**. DO NOT post duplicates
- [X] I've read the [guidelines for opening an issue](https://github.com/yt-dlp/yt-dlp/blob/master/CONTRIBUTING.md#opening-an-issue)
- [ ] I've read about [sharing account credentials](https://github.com/yt-dlp/yt-dlp/blob/master/CONTRIBUTING.md#are-you-willing-to-share-account-details-if-needed) and I'm willing to share it if required

### Region

Ireland

### Provide a description that is worded well enough to be understood

This plays fine in a browser in Incognito mode, so it's not related to the only other Twitter bug I found, which was about Twitter requiring login.

### Provide verbose output that clearly demonstrates the problem

- [X] Run **your** yt-dlp command with **-vU** flag added (`yt-dlp -vU <your command line>`)
- [ ] If using API, add `'verbose': True` to `YoutubeDL` params instead
- [X] Copy the WHOLE output (starting with `[debug] Command-line config`) and insert it below

### Complete Verbose Output

```shell
[debug] Command-line config: ['https://redd.it/15zix69', '-vU']
[debug] Encodings: locale UTF-8, fs utf-8, pref UTF-8, out utf-8, error utf-8, screen utf-8
[debug] yt-dlp version stable@2023.07.06 [b532a3481] (zip)
[debug] Python 3.8.10 (CPython x86_64 64bit) - Linux-5.4.0-156-generic-x86_64-with-glibc2.29 (OpenSSL 1.1.1f  31 Mar 2020, glibc 2.31)
[debug] exe versions: ffmpeg 4.2.7, ffprobe 4.2.7
[debug] Optional libraries: Crypto-broken 2.6.1, certifi-2019.11.28, secretstorage-2.3.1, sqlite3-2.6.0
[debug] Proxy map: {}
[debug] Loaded 1855 extractors
[debug] Fetching release info: https://api.github.com/repos/yt-dlp/yt-dlp/releases/latest
Available version: stable@2023.07.06, Current version: stable@2023.07.06
Current Build Hash: cedf44a9cf3c241520a692c1006d276e6f434d4d945727c47cb1a70d32953f4a
yt-dlp is up to date (stable@2023.07.06)
[generic] Extracting URL: https://redd.it/15zix69
[generic] 15zix69: Downloading webpage
[redirect] Following redirect to https://www.reddit.com/comments/15zix69?rdt=48688
[Reddit] Extracting URL: https://www.reddit.com/comments/15zix69?rdt=48688
[Reddit] 15zix69: Downloading JSON metadata
[twitter] Extracting URL: https://twitter.com/wendoverpro/status/1694473951987048573?s=20
[twitter] 1694473951987048573: Downloading syndication JSON
[twitter] Some metadata is missing without authentication. Use --cookies, --cookies-from-browser, --username and --password, --netrc-cmd, or --netrc (twitter) to provide account credentials
ERROR: [twitter] 1694473951987048573: No video could be found in this tweet
  File "/usr/local/bin/yt-dlp/yt_dlp/extractor/common.py", line 710, in extract
    ie_result = self._real_extract(url)
  File "/usr/local/bin/yt-dlp/yt_dlp/extractor/twitter.py", line 1276, in _real_extract
    self.raise_no_formats('No video could be found in this tweet', expected=True)
  File "/usr/local/bin/yt-dlp/yt_dlp/extractor/common.py", line 1188, in raise_no_formats
    raise ExtractorError(msg, expected=expected, video_id=video_id)
```

**Comment by bashonly** [comment]
Duplicate of #7850

Fixed by 92315c03774cfabb3a921884326beb4b981f786b

If you don't want to wait for next stable release, run `yt-dlp --update-to nightly` or [install the master branch](https://github.com/yt-dlp/yt-dlp/wiki/Installation#pip-master)

**Comment by TimothyTRiG** [comment]
Thanks. I did search; not sure how I missed that one.
